src/mz/Readme.md

```diff
@@ -0,0 +1,127 @@
+# Materialize CLI [WIP]
+
+Rust implementation of critical features for usage and deployment of Materialize platform instances.
+It provides the following actions:
+
+* Login
+* Enable/Delete Regions
+* Shell
+
+## Documentation
+
+### Install
+
+Clone the repository. Once inside, run:
+
+```bash
+cargo build --package mz --release
+```
+
+After a succesfull build. Make the binary executable:
+
+```bash
+chmod +X ./target/release/mz
```

Comment:
This is surprising! It should already be marked as executable.

```suggestion
```

Reply:
Super. Replacing with `environmentd:latest`